### DISCUSSION TITLE
Q2S1 (April to Mid-May) Roadmap Planning

### DISCUSSION THREAD

**Discussion by BenJam** [opencollective/opencollective#4161] [Ideas]
It's the start of a new quarter! Which means it's time to prioritise for the next sprint. 

Given that this week is a bit piecemeal with public holidays and suchlike we only have a small amount of time we can spend synchronously, as a result we need to move as much of this process as possible to a more asynchronous conversation, thus this _discussion_. 

My intention here is to wrap a little guiadnace around a process that maintains our democratic approach to prioritisation and leaves the space for us to contribute to and discuss each of the projects that we would like to focus on. 

## What is this discussion for?
Prioriting [projects](https://github.com/opencollective/opencollective/issues?q=is%3Aopen+is%3Aissue+label%3Aproject) and other work that Open Collective will dedicate our time to weeks commencing Mon 12th to May 3rd. 

## How do I contribute?
I'm going to propose some minimum process here, most of which rely on using the threaded comments structure of Discussions:

**Proposing a new project**
Create a top-level comment introducing your idea. Consider creating a new project-level issue for it (or at least using that to organise your thoughts). Make sure you include your case as to _why_ we should commit ourselves to the idea, what benefit you expect to see and how  we could measure our success.

**Proposing a project that has already been scoped**
Create a top-level comment introduce your case for a project. Link to the issue(s) that are appropriate. Make sure that the issues asnwer the _why_ that we have for proposing a new project but pay extra attention to _why now_. Consider `@mention`ing the people who created the original issues to get their opinion. 

**Proposing something that is not a 'project'**
Is completely legitimate too, just create a top-level comment for it. Try to ask yourself the same questions so that others can compare it with time we might dedicate to projects: 'why this/these thing(s)?', 'why now?', 'what do we expect to see a result of this work?' and 'how will we gauge our success?' are all valid questions to consider. 

**Discussing others' ideas**
Feel free to lend thoughts, opionions, data and ask questions in threads from each of the top-level comments (which should contain project). 

**Voting**
We dont _have_ to do this but feel free to 👍 any project ideas that you feel would be a good fit for this sprint. Try to limit yourself to three votes for the sprint to keep things focussed. 

## Our Priorities
It's obvious but worth repeating that we should prioritise projects that align with our mission _and_ our priorities at the time we commit to a project. At the Open Collective team 'retreat' we reaffirmed our mission and identified our priorities for Open Collective in 2021:
> Our goal is to catalyze impact by enabling communities to have economic power, providing them with tools to be financially independent and unlock access to funding.

Our priorities for 2021 are about deepening our core strengths: 

* Building up our own non-profit Fiscal Hosts and scaling up adoption by third party Fiscal Hosts (with a shared revenue model.)
* Facilitating more contributions to Collectives
* Facilitating disbursement of funds.

You can read more in [Pia's investor update](https://blog.opencollective.com/2020-in-review-2021-update/).

## Next steps

**Synchronous discussion**
We'll arrange a synchronous discussion for everyone Thursday 8th to talk through the top pitches for the quarter and to create the space for us to decide together based on the outcome of this discussion. 

**Kickoffs and further discussion**
This plan leaves a small amount of time for kick-offs to happen on Friday, groups will need to arrange these together.

**Comment by BenJam** [comment]
Attempting to lead by example, I'd like to propose my first pitch:

## [Standardising feedback processes, tooling and guidance](https://github.com/opencollective/opencollective/issues/4134)

>To get more value out of the product process we’re need invest a little time in building a standard process and tools for releasing new features, directing users to giving feedback and measuring our success.

I do not think this is a heavyweight project that we need to commit a team to for the entirety of the sprint, it's likely that it will predominantly involve my time and some design time to standrdise some components for calling a new feature to attention and dropping the user into the feedback flow we decide upon. 

I think this project could go well with @Memo-Es's proposed changelog project https://github.com/opencollective/opencollective/issues/4138 maybe combining the two might be meaty enough for us to do as a project. 

** Why should we do this? **
As mentioned in the linked issue for me this starts the wheels turning on a whole load of other processes that will ensure we're building the right product, for the right audience, at the right time. It'll force us to be clearer about what is in our roadmap and more importantly it will force us to rationalise why we're not committing six months to _that feature you need right now_. It will unlock a more _engaged_ userbase who will help us build the future of open collective and it will expand our outreach by making them our biggest fans. 

** Why now?**
Open Collective has just hired a fresh faced head of product! And we'll get more out of him if create the patterns needed to establish a product lifecycle that _anyone_ in the team can later use to build a better Open Collective 🙌

**Reply by BenJam** [reply]
feel free to discuss in this thread why we should/should not commit to this project this sprint. If you have another project you'd like to pitch start a new comment and we can discuss that in another thread. Thanks!

**Comment by alanna** [comment]
I'm assuming it's best to do one idea per comment so we can be organised with threading? I'll do that for mine.

**Comment by alanna** [comment]
## Events

I wrote about why it could be a good project to take on in this issue #4129 - basically, it's a feature used and loved by users, but has not been loved by us enough. There are so many low hanging fruit that would drastically improve events, so even just a smaller project to do those without touching any hard stuff would be a big step up.

**Reply by alinamanko** [reply]
Would love to see it happening. I think some simple changes in this space would make a world of a difference to the user experience.

**Reply by Betree** [reply]
I'm 100% for this project, but as mentioned in https://github.com/opencollective/opencollective/issues/4129#issuecomment-809146282 I think we should have a session to properly scope the issues that we want to see fixed for this sprint. Depending on the ones we choose, the complexity can go from one day to one month. It's really difficult for me to prioritize something if I have no idea of how long it's going to be.

**Comment by alanna** [comment]
## Complete the pricing migration

We've already committed to moving every host still on the old plans over on 1 May. So we have to work on it this sprint. We should properly consider all the things that will need to be in place for the migration, including comms, docs, pricing page, settings updates, fixing host metrics, etc, so everything can be switched over at the same time.

Related: #4056 #3927 #3917 #3592 #3381 #4123 #3652 #4060

**Reply by BenJam** [reply]
I think this is very much 'committed' and something we pretty uch have to do, no?

**Comment by alanna** [comment]
## Crypto

We've been talking in team meetings a lot about crypto and researching some potential solutions like Giving Block, but last time it came up we agreed if we're going to work on it we should do it as a proper prioritised project and do a thorough job of comparing options and designing the right implementation. I think it's a huge opportunity and would have a big impact for hosts like OSC and OCF.

**Reply by BenJam** [reply]
IOU an issue for this and I'll pull one together in the morning but as far as I understnad it we're looking at using a partner to accept crypto an convert it immediatly into the fiat currency of the collective yes? 

If that's the case then I'm all for it. What I'm not a fan of is the idea of holding crpyto blanaces for projects, operating wallets and generally supporting all of the rather-more-complicated-than-actual-money use cases.

**Reply by alanna** [reply]
> What I'm not a fan of is the idea of holding crpyto blanaces for projects, operating wallets and generally supporting all of the rather-more-complicated-than-actual-money use cases

Yes, this would be an accounting nightmare. But some services may take care of all that for us? Some scoping would need to be done as part of this project. 

But the core spec is: 

* Find the right exchange partnership
* Be able to accept multiple currencies, especially ETH in addition to BTC
* integrate the crypto option to the checkout flow
* automate crediting of fiat that was exchanged from crypto

Maybe the right project to pitch for this roadmap point is actually just scoping and research, and treat implementation separately.

**Comment by alanna** [comment]
We also have the [issues with the `important` tag](https://github.com/opencollective/opencollective/issues?q=is%3Aopen+is%3Aissue+label%3Aimportant) to consider. This tag doesn't really have a clear rubric for when it's used, but we should look at them at least.

**Comment by alanna** [comment]
## Bug fixes and paper cuts

I still feel like it would be worthwhile to set aside some capacity (like a week) for the team to smash as many small issues as possible, and also engage our wider community of bounty contributors (especially those who have done successful bounties already) in that push. We tend to always focus on new projects over the many small improvements that would make the platform more polished and a better overall experience. I think treating such a push as a project might be the best way to weave it into how we allocate team time and prioritisation.

**Reply by BenJam** [reply]
I would ver much like for us to plan each sprint to have some time for papercuts, whether those are things that have been annoying _us_ (unnecessary warnings, flakey tests etc) or users (see feedback process). This was one of the most celebrated initiatives at GitHub both inside and outside of the walls of the organisation and it's a great way to engage users, enthuse them and make them feel a part of the team.

**Reply by Betree** [reply]
I think this has proven to be beneficial in the past when we did it. There are a lot of small bugs and annoying issues that we can fix in one week.

**Comment by alanna** [comment]
Slightly tangential but in the lead up to the next quarterly roadmap process (not this time), it would be cool to send a survey to host admins to get their input or something like that.

**Comment by piamancini** [comment]
Widgets for fundraising: one of the goals of this year is to help collectives raise more funding. We want to build widgets for collectives to be able to share, use on their own pages, etc. to better fundraise.

**Reply by alanna** [reply]
How has uptake been of the embed contribution flow feature? That could give us a clue about demand.

**Reply by piamancini** [reply]
@Betree knows more about this. But we've slowly added some collectives to the beta. There's certainly demand.

**Reply by Betree** [reply]
The two places to look for examples and feedback are [Slack](https://opencollective.slack.com/archives/C01NGE0N9U1) and [this spreadsheet](https://docs.google.com/spreadsheets/d/1F3i75EXEsynQlNxYPTrtLhfi-nwW5VXFF7eSp916BEw/edit). Here are [two](https://www.xrwestnorfolk.org/donate/) of [them](https://affecttheverb.com/donate/) that IMO look pretty nice.

We only communicated about the embed contribution flow in https://github.com/opencollective/opencollective/issues/3003 and in the newsletter, yet we already had ~20 collectives joining the beta. To me, it's a good sign that there is demand for it.

I think the next step is to release it as an official product: add it to the docs, to the `/collective/edit/export` page, post announcements, etc.

After that (or in parallel), if we want to go further with this project, we can start thinking about a real dedicated widget (not just an embedded version of the existing contribution flow). Something small & simple, kinda like what you have on https://signal.org/donate/.

**Comment by piamancini** [comment]
Better integrate with Github: we are still doing this process very manually and there's an opportunity for us to better manage this

**Reply by BenJam** [reply]
I know GitHub are looking to improve the support for hosts like Open Source Collective etc. but how much can we self serve vs. relying on them? 

For me there are two sieds to this: how open collective processes transactions and integrates with the back office  (which I think is what you're proposing here) and how the integration works for users using GItHub with open collective i.e. the registration process. 

I would like to pitch a project tackling the l;ater because I know we have multiple issues with the signup flow on GitHub. Effectively I'd like to make the open colelctive part invisible and lower the barriers to entry for new projects: using a pre-registration state that will hook projects up to sponsors and enable them to start collecting sponsorship before we validate them.

**Reply by alanna** [reply]
There are potential projects for both our Sponsors integration and our signup flow integration with Github. Unless the proposal is to integrate those together (never considered this before! it could be awesome but I assume coupling them creates too many complexities), we should treat them as two separate projects.

Personally I think all the biggest pain points for end users (maintainers/Collectives) are pretty much out of our control and in Github's court - this mainly is about the setup process. If we want to work on anything to do with Sponsors, we need to carefully scope it to stuff either we can do without them, or get some commitments in writing from them about how/when they will do their part so doing our part isn't a waste of time.

I think a huge win would be Github integrating the fiscal hosting option into the Sponsors signup flow, but that's pretty much out of our control. Alyssa has been working with Alyss on the comms side of that.

**Reply by BenJam** [reply]
> I think a huge win would be Github integrating the fiscal hosting option into the Sponsors signup flow

What exactly do you mean by this, is it scoped elsewhere? /cc @piamancini ?

**Reply by alanna** [reply]
Alyssa and I have discussed it with them extensively. Basically if someone is trying to sign up for Sponsors for Organizations, they need a corporate bank account. If they get to that point and they don't have one, they should be informed that signing up with a fiscal host is an option to solve that for them. Ideally, since OSC is the only one they can actually sign up with instantly, they could click something there to create their Collective and not be blocked.

**Comment by piamancini** [comment]
Finish & deploy Virtual Cards

**Reply by alanna** [reply]
We decided to first deploy this only to the opencollective host for testing. Is that a good goal for this sprint or are we talking about going all the way to full rollout?

**Reply by piamancini** [reply]
It's pretty much done for testing internally so the scope would be to roll out

**Reply by alanna** [reply]
If we choose to prioritise this one I would like to be involved from a compliance perspective. We need features like freezing their card if they don't upload their receipts.

**Comment by piamancini** [comment]
Open Stocks integration, launch and management

**Reply by alanna** [reply]
We've already talked about this publicly and started far down the track of launching it, so I think we have to commit to seeing it through.

**Comment by piamancini** [comment]
Finish & deploy PayPal recurring

**Reply by alanna** [reply]
This is another one we've already done a lot on and I think we have to prioritise to get it launched.

**Reply by Betree** [reply]
I agree. The public release is coming soon, but we'll surely have small things to iterate on.

**Comment by kewitz** [comment]
### Ledger Refactor

The ledger is getting a bit behind compared to the features and business model updates we've been shipping.
@znarf and I discussed a bit about some refactors we would like to how we record transactions to increase consistency and solidify some of the changes we made in the past year.
Some of the things we want to make here include:
- Add separate entries for each fee
- Track settlement status per transaction

**Reply by alanna** [reply]
I don't really understand what this means, but 'ledger debt' has really haunted us before and we should not neglect it just because it's more behind the scenes.

**Reply by piamancini** [reply]
This can be a huge project but it's necessary and there has been progress done already in scoping.

**Reply by Betree** [reply]
Now that the technical part has been scoped, I also think that we should prioritize it. If we don't do it now we'll have to build on technical debt and we'll pay for it later.

**Reply by znarf** [reply]
The issue is https://github.com/opencollective/opencollective/issues/4124

**Comment by piamancini** [comment]
Scope a project to enable collectives & funds to do grantmaking

**Comment by Memo-Es** [comment]
## A better discovery experience to find hosts (Host-shopping)

We have the design for the new marketing page 'Fiscal Hosting as a Service.' Still, we have many opinions from hosts that tell us it's not enough to help Collectives find the right host. By improving that experience, we will scale the adoption of the whole 'fiscal host' concept.

**Reply by kaylarep** [reply]
#3892

**Reply by BenJam** [reply]
Agreed we could do a lot more here. One of the things I have been thinking is that as we foreground hosts and background open collective the open colelctive site itself should become a signposting site. 

> 'Want to start a climate action community based in your town? here's a host that can support you' 

should be a flow that the site supports well, equally

> Looks like we haven't got a host set up in your country that supports climate initiatives. Why not set one up?

Is a very easy flow to get users into if they can't find what they're looking for.

**Reply by piamancini** [reply]
+1 to this - it also goes to similar collectives to support. I very much like the cart checkout idea in app.fundoss.org

**Reply by alinamanko** [reply]
Absolutely!

**Reply by BenJam** [reply]
I feel like this, along with the marketing materials that @aminakazi has proposed reviewing below would make for a good product+design project this sprint, just to scope out the changes and build something that we can then implement soon after.

**Comment by natehn** [comment]
### Partial Deductibility

Note: this is probably only an issue for US-based 501(c)(3)s, but I wanted to put it on your radar. 

Partially deductible donations are when 501(c)(3)s incentivize donations of a certain level by offering benefits in return for the donation. Crucially, the value of goods/services received cannot exceed the amount of the donation. When such an exchange occurs, the donor’s tax receipt states the full donation amount, the market value of goods/services received, and the amount that is tax-deductible for the donor. 

The tax-deductible amount is equal to the difference between the total donation and the value of the goods/services received. For example, for a $50 donation where they receive a $10 t-shirt, $40 is tax-deductible.

US 501(c)(3)s are legally able to receive partially-deductible donations, but the OC platform is not set up for it. If we were to add such a feature, we would do the following:

- For Tiers, add the option for collectives to include a market value/non-deductible amount and information about the rewards donors would receive
- The system would use that amount on the backend to automatically adjust the generated tax-receipts accordingly
- EDIT: we might also want to collect mailing addresses of donors, if the reward is in the form of a physical good of some sort

**Reply by piamancini** [reply]
How big of an issue is this for OCF and how it's replicable to other hosts? 
Is this something OCF is willing to fund?

**Reply by alanna** [reply]
To me it's related to ecommerce as well, basically do we want to support Collectives selling stuff as a means of fundraising. That function is often requested, but it's likely to only result in small amounts of money coming in IMO. But if we tell Collectives they can sell stuff, we will immediately face all the other issues related to our lack of ecommerce features, like handling sales tax, collecting a buyer's shipping address as part of the process, etc. I think it's all too complex and too much of a distraction to prioritise right now. That said, I actually think we could design a very MVP version of a feature for this super easily. It would just be a setting on a tier where you input the item value, then it would display some extra data on the receipt and that's it. So technically it would be super simple, but the non-technical aspects are not simple at all. It's something I hope we can take on in the future but to me doesn't seem like the most urgent need for OCF. But I'm open to @natehn and @kaylarep disagreeing with that!

**Reply by alanna** [reply]
If we ever do decide to prioritise ecommerce as a function, we should look into integrating with a third party provider, and at that point, we could make partial deductibility an aspect of the implementation.

**Reply by BenJam** [reply]
I can see collectives outside of Open Source Collective wanting to do this, but yeah... the implication on administrative overhead are _not small_ https://twitter.com/naderman/status/1379745426367217665?s=21

**Reply by natehn** [reply]
I definitely understand that e-commerce would be wayyy too much right now (and maybe - probably? - too much, period). But this is something different in the US - there's just a superficial similarity in that goods and services are changing hands. We wouldn't be telling collectives that they can sell things, but rather that they can offer rewards, which is legally and logistically different from selling things for 501(c)(3)s in the US. We also would not need to collect taxes since it is not a sale - again, this is in the US. But I hear you on collecting mailing addresses. While not necessary for the bare-bones version of this feature, it would be helpful for collectives who might offer rewards in the form of physical items.

I suspect that the issue GitHub seems to be having, described in detail in the thread @BenJam shared, might have to do with the fact that they are US-based. The same issues might crop up for OC if non-US hosts had access to a "partial deductibility" feature, since it seems to be somewhat unique to the US tax code (note - I don't have a ton of knowledge about non-US tax law so don't know this for sure). As a result, I understand that this won't be a priority unless OCF decides to fund it (which I don't think we're ready to do yet as we consider OCF's long-term strategy).

Anyway, I can see that this is more of a feature request - should I make an issue for this instead for continued conversation?

**Reply by kaylarep** [reply]
I would +1 having an option for our groups to sell merch to fundraise in some way, and +1 having it the simplest way possible

**Comment by Memo-Es** [comment]
## Admin panel implementation (Settings update)

We did a full update to the settings of users, collectives, organizations, and fiscal hosts last year when we did the host dashboard.
@Betree and I think it is time to get it done with a proper project. The design is practically ready, but it will need some design time to make sure everything is updated and accompany the developer process.

**Reply by piamancini** [reply]
yes - this was a case of design and then in the future find the time -
What would the biggest pain points for our users we would be solving with it?

**Reply by Betree** [reply]
We now have a dev proof-of-concept for this one. On the engineering side, it's not super expensive; most of the work is things we need to iterate on is in sync with design. It's important to note that we've been postponing this for some time, which proves it's not an emergency.

Now could be a good time to prioritize it because:
- It will lay the ground for other features to come like the budget page or https://github.com/opencollective/opencollective/issues/3894 - no more "where should I put this?"
- It will create a better experience for both dashboard and settings. The current pages both look a bit old.
- It could simplify the host onboarding experience
- It's (theoretically) not super expensive to do

**Reply by alanna** [reply]
This doesn't seem super urgent, but it's definitely aligned with the strategic focus on supporting thriving hosts. I feel like I'm the worst person to judge how painful the current experience is because I am so used to its quirks and the workarounds, etc. The real test is do new admins engage well with the settings.

**Reply by piamancini** [reply]
I think that if it lays the groundwork for the standalone budget page then it's worth doing. The page has metrics which is something collectives (and hosts) have been asking for a while

**Reply by BenJam** [reply]
if we're so far forward with the design that it kinda falls into the realms of 'implement this' might this be a candidate for bringing someone in temporarily to help us implement it?

**Comment by Betree** [comment]
# Better protect against fraudulent credit card transactions

**Why should we do this?**
We have an increasing number of fraudulent credit card transactions coming from guests, which is a real problem as it translates to important fees for hosts. @znarf added some smart & simple protections to fool the fraudsters, but they can easily be bypassed and we're clearly not ready for more intensive attacks.

We don't have yet a project to specify how we want to tackle this, but steps for this project could be things like:
- To provide more data (device data, name, etc) to Stripe to help them identify attackers
- Introduce captcha to prevent automatic attacks
- Analyze the traffic & see how we could to block them

**Why now?**
Being taken by surprise (if the attacks get more clever) and having to iterate in the rush is a risk of coming up with poor solutions, and spending more time to iterate on them - compared to implementing something well-designed and robust from the start. The attacks are happening as we speak, so whether we prioritize it or not we will certainly end up working on this.

**Comment by Betree** [comment]
# [Improve the accessibility of the platform](https://github.com/opencollective/opencollective/issues/4045) (@Memo-Es)

**Why should we do this?**
- By improving the accessibility of the platform, we are being more inclusive
- As a side-effect of this change, we end up improving things like readability for everyone
- It's also solving some related UX issues, like https://github.com/opencollective/opencollective/issues/3528

**Why now?**
- This project is pretty simple. Most of the tasks can be done quickly or outsourced as bounties
- As a result of the previous, this project will help us reach our bounties target (I think it was ~100 bounties created / year?)
- Designs are ready. If we tackle this project later, there's a risk that the designs will be outdated and some work will thus be lost

---

For me, a nice goal for this project would be to limit at the maximum the engineering time directly invested (should be less than a day) and outsource everything else as bounties.

**Reply by BenJam** [reply]
Is there a group of issues that we want to target specifically? accessiblity is always a concern and a priority. I wonder whether we can start to build this into our development process by not considering something 'complete' until it is accessible, it is documented and it has a channel for feedback :D

**Reply by Betree** [reply]
> Is there a group of issues that we want to target specifically?

For this sprint, I was thinking about tackling everything in https://github.com/opencollective/opencollective/issues/4045. It's mostly about updating some states in the design system, some hover states, UI. Mostly CSS.

> I wonder whether we can start to build this into our development process by not considering something 'complete' until it is accessible, it is documented and it has a channel for feedback :D

Definitely. We increasingly progressing toward that, we've [recently added](https://github.com/opencollective/opencollective-frontend/pull/5961) accessibility checks to our CI.

**Comment by aminakazi** [comment]
### Reformat our marketing pages to better explain our position in the market

**Why should we do this?**
1. Our current landing page doesn't clearly explain our USP. It is more oriented towards donation.
2. People are unaware of our features and services. 
3. There is a broken trail between pages because they are not interlinked with each other (For e.g Home Page > Fiscal Hosting as a Service > Become a fiscal host) in the most effective manner. 

**Why now?**
1. We can solidify out position in the market by reformating the design + content of our homepage to highlight our best features. 
2. With the addition of pages like Fiscal Hosting as a Service, How it works etc, it is a good time to re-align the pages and sections to interlink them effectively.

**Reply by BenJam** [reply]
this feels like something we can do without needing significant development time this sprint. I think it might also be a great time to do this as we begin to foreground hosts and background the platform with the whitelabel/host as a service work.

Incidentally I dont see any of that whitelabelling work being discussed here... is that something that we've missed?

**Reply by piamancini** [reply]
yes - whitelabeling is missing. I'll add it

**Reply by Memo-Es** [reply]
Refresh our marketing pages is something that we have been talking about superficially the past few weeks, but I think that it will turn out to be a big project. Amina describes turning our current homepage into the page for Collectives, which sounds easy, but that also entails designing a new homepage, one from an organizational perspective that covers all the stuff we offer to the different communities and projects we work with.
I also think that this project is related to the idea of having different experiences and interfaces for logged-in and not logged-in users.

I suggest taking the proper time to think about this project later on in the quarter :)

**Reply by aminakazi** [reply]
+1 for what @Memo-Es said. The project will mean a good amount of reformation of our homepage.

**Comment by piamancini** [comment]
## White-labeling

Work on the idea of white-labeling and mature it to feed more projects in the future.

**Reply by piamancini** [reply]
SCA came back to us with the following: 

> 
> "Looking at the effort it would take to implement it though, I'm not sure it's the biggest priority within our list of needs and budget allocation. I had gotten the impression that it was already built and available through OC Foundation, but if it's a new custom flow I appreciate there are more efforts involved and you need the resources to do it. Is this going to be the case for everything going forward? 
> 
> Is there a way we could implement just the step on page '4. Application Form'? And in the spirit of open source and the community, if we get this custom feature can it be made available to other hosts? "

**Reply by Memo-Es** [reply]
I sense that this project also feeds a couple of ideas like https://github.com/opencollective/opencollective/discussions/4161#discussioncomment-583222.
Aside from following up with SCA, I feel that we could do some work to mature a little bit the whole white-labeling idea

**Comment by znarf** [comment]
### Projects

Users are asking to setup Projects for their Collectives (it's currently only available for Funds). 

It seems we're willing to give them this.

**Reply by piamancini** [reply]
https://github.com/opencollective/opencollective/issues/4074

**Comment by Memo-Es** [comment]
## Changelog + What's new? modal

We have this project scoped already in https://github.com/opencollective/opencollective/issues/4138, and it is related to what @BenJam suggests in https://github.com/opencollective/opencollective/discussions/4161#discussioncomment-575219. 

I totally see a tandem project in there for this sprint 🤓
